bors.toml

```diff
@@ -0,0 +1,4 @@
+status = [
+  "continuous-integration/travis-ci/pr"
```

Comment:
This line is not what you want.

What you probably want is `"continuous-integration/travis-ci/push"`, [like this](https://github.com/nix-rust/nix/blob/master/bors.toml#L2).

Reply:
thanks I have tried but it's still timeout and everything is configured as explain in the FAQ.

Reply:
I'm not sure why everything is timing out, but I can still tell you that it needs to be the Push one, not the PR one. The PR one will not work.